examples/combined-openapi-mock/app.js

```diff
@@ -4,7 +4,7 @@
 const openapiApp = require("@varkes/openapi-mock")
 const connectorApp = require("@varkes/app-connector-client")
 const app = require('express')()
-
+const cockpitApp = require("@varkes/cockpit");

```

Comment:
please move it up to the other varkes packages

Reply:
done